newm-server/src/main/kotlin/io/newm/server/features/cardano/repo/CardanoRepositoryImpl.kt

```diff
@@ -272,6 +274,26 @@ internal class CardanoRepositoryImpl(
         }
     }
 
+    override suspend fun verifySignData(
+        signatureHex: String,
+        publicKeyHex: String
+    ): VerifySignDataResponse {
+        return client.verifySignData(
+            verifySignDataRequest {
+                this.signatureHex = signatureHex
+                this.publicKeyHex = publicKeyHex
+            }
+        )
+    }
+
+    override suspend fun verifySignTransaction(signatureHex: String): VerifySignDataResponse {
```

Comment:
This should be passing the entire transaction's cborHex that has been merged with the signature, not just the signature itself.

Reply:
Yes, I intended that. I'll rename the field to make it clear